### ISSUE TITLE
Double speed

### ISSUE THREAD

**Issue by christantoan** [chiehmin/Xposed-NetworkSpeedIndicator#111]
Showing double speed when using VPN or proxy

**Comment by portugt** [comment]
Same problem 🤔

**Comment by ShapeShifter499** [comment]
@chiehmin I'm seeing this same issue, I'm using a VPN.

My Device:
Motorola X Pure
Android 6.0.1 Cyanogenmod 13 Nightly 11-21-2016
Xposed v0.87-test1

**Comment by yjmp14** [comment]
Yes, I have the same problem. The App Internet Speed Meter doesn't have this issue, but that app doesn't show the second SIM card's network speed.

**Comment by ShapeShifter499** [comment]
Any updates on this @chiehmin?

**Comment by seasonyuu** [comment]
@ShapeShifter499 build the latest version would solve this problem , and the version of apk in the [release](https://github.com/chiehmin/Xposed-NetworkSpeedIndicator/releases) is too old to use.

**Comment by ShapeShifter499** [comment]
@seasonyuu I don't have any build environment setup. I haven't compiled any apps for Android yet.
